### PR TITLE
update cs3apis

### PR DESCRIPTION
**Pull request by micbar**
# Description

Update CS3 apis to latest.

We currently do not implement `ListTransfers` and `RetryTransfer`

Is that already implemented on `master`? @ishank011 @labkode can you clarify?

Let us go forward here and try to merge master into edge.

### CODE REVIEW

File: internal/grpc/services/gateway/storageprovidercache.go
Comment (micbar) [inline internal/grpc/services/gateway/storageprovidercache.go:235]: This fixes a bug in the cache

**Review by wkloucek — APPROVED** [review]

**Comment by glpatcern** [comment]
> Is that already implemented on `master`? @ishank011 @labkode can you clarify?

That is pending https://github.com/cs3org/reva/pull/2052, which was pretty much ready. @ishank011 I guess we have to prioritize the re-review.

**Comment by micbar** [comment]
> > Is that already implemented on `master`? @ishank011 @labkode can you clarify?
> 
> That is pending #2052, which was pretty much ready. @ishank011 I guess we have to prioritize the re-review.

ok. I will postpone this PR until we merge #2052

Let us speed this up and merge master into `edge` as soon as possible. The challenge here is that all further CS3Api changes are blocked by this.

**Comment by wkloucek** [comment]
cs3api was already updated

**Pull request closed without merging**
